src/table/components/__tests__/TableWrapper.spec.jsx

```diff
@@ -111,10 +113,45 @@ describe('<TableWrapper />', () => {
         keyboard={keyboard}
         translator={translator}
         rect={rect}
+        announcer={announcer}
       />
     );
+
     fireEvent.change(getByTestId('select'), { target: { value: 25 } });
+
     expect(setPageInfo).to.have.been.calledWith({ page: 0, rowsPerPage: 25 });
+    expect(announcer).to.have.been.calledOnceWith({
+      keys: [['SNTable.Pagination.RowsPerPageChange', sinon.match.any]],
```

Comment:
same here

Reply:
actually make 25 into a const, so we see that the value we fire is the same as `setPageInfo` and `announcer` is called with

Reply:
✅ Fixed